[3.3] docs(dpl): Update feature list (#929)
telerik/blazor-docs: components/document-processing/spreadsheet.md

* **[Auto fill](https://docs.telerik.com/devtools/document-processing/libraries/radspreadprocessing/features/fill-data-automatically/repeat-values) and [Series](https://docs.telerik.com/devtools/document-processing/libraries/radspreadprocessing/features/fill-data-automatically/series)**: Fill cells automatically with data following a specific pattern.

* [**Page Setup**](https://docs.telerik.com/devtools/document-processing/libraries/radspreadprocessing/features/worksheetpagesetup): Set and get header and footer settings and apply various page setup options like paper size, orientation, scaling, margins, breaks, etc. Apply print settings.

* [**History**](https://docs.telerik.com/devtools/document-processing/libraries/radspreadprocessing/features/history): The document model provides the possibility to maintain a history stack that tracks all changes to the content of the workbook. Each worksheet has its own history stack.

> * [**Comments**](https://docs.telerik.com/devtools/document-processing/libraries/radspreadprocessing/features/comments): You can leave comments and replies on cells, which allows for easier team communication.

Author: ```suggestion
* [**Comments**](https://docs.telerik.com/devtools/document-processing/libraries/radspreadprocessing/features/comments): Leave comments and replies on cells, which enables easier team communication.
```

> * [**Notes**](https://docs.telerik.com/devtools/document-processing/libraries/radspreadprocessing/features/notes): Note important information in your worksheet.

Author: ```suggestion
* [**Notes**](https://docs.telerik.com/devtools/document-processing/libraries/radspreadprocessing/features/notes): Annotate and clarify information in worksheet cells.
```
